repair_order_template/models/repair_order_template.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Camptocamp SA (https://www.camptocamp.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class RepairOrderTemplate(models.Model):
+    _name = "repair.order.template"
+    _description = "Repair Order Template"
+    _check_company_auto = True
+
+    name = fields.Char(required=True, translate=True)
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one("res.company", default=lambda self: self.env.company)
+    tag_ids = fields.Many2many("repair.tags", string="Tags")
+    internal_notes = fields.Html()
+    under_warranty = fields.Boolean()
+    product_id = fields.Many2one(
+        "product.product",
+        string="Product to Repair",
+        domain="""
+            [
+                ('type', 'in', ['product', 'consu']),
+                ('company_id', 'in', [False, company_id])
+            ]
+        """,
```

Comment:
This could be directly set as a `list` instead of a `str`
```suggestion
        domain=[('type', 'in', ['product', 'consu']), ('company_id', 'in', [False, company_id])],
```